### PR TITLE
Pre-commit integration

### LINKED ISSUE #14: Configure pre-commit tool for this repo
> # Rules:
> 1. **DO NOT** commit to master. ever.
> 2. **ABSOLUTELY NO** force pushes allowed for public non-feature branches.
> 3. For every feature, create a new branch matching name `feature/<issue-number>-<feature-name>`.
> 4. Submit every feature by creating Pull Request @ github, after that wait for review, fix issues, repeat until approved. If needed rebase the branch on top of master and re-push force it.

  * [x] **I HAVE READ AND UNDERSTOOD THE RULES ABOVE**
------------

* [x] Install pre-commit locally as follows:
```
python3.7 -m pip install pre-commit --user
python3.7 -m pre_commit install
```
* [x] Create branch called `feature/pre-commit-integration`
* [x] Add [`.pre-commit-config.yaml`](http://pre-commit.com/) config, containing the following [hooks](http://pre-commit.com/hooks.html):
  * [x] no-commit-to-branch
  * [x] [add-trailing-comma](https://github.com/asottile/add-trailing-comma)
  * [x] [remove-tabs](https://github.com/Lucas-C/pre-commit-hooks)
  * [x] trailing-whitespace
  * [x] check-merge-conflict
  * [x] double-quote-string-fixer
  * [x] requirements-txt-fixer
  * [x] check-yaml
  * [x] check-docstring-first
  * [x] end-of-file-fixer
  * [x] name-tests-test
  * [x] check-ast
  * [x] debug-statements
  * [x] flake8
  * [x] [pydocstyle](https://github.com/chewse/pre-commit-mirrors-pydocstyle)
  * [x] `pylint` using "local" mode
  * [x] [mypy](https://github.com/pre-commit/mirrors-mypy.git)
* [x] Check repo using command `pre-commit run --all-files`, fix all issues and commit fixes as well
* [x] Create PR with this branch

### PR DESCRIPTION
**Pull request by RootenberG**
Closes #14

### CODE REVIEW

**Comment by webknjaz** [comment]
Why do you keep closing PRs? Just fix them.

**Comment by RootenberG** [comment]
@webknjaz Why when I created a pool request in it immediately appeared an old commit with the Travis file?

**Comment by webknjaz** [comment]
PR just represents what is the difference between your feature-branch and the target branch. If you need a different result, change your feature branch.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+language: python
```
Comment (webknjaz) [inline .travis.yml]: Let's only have related changes in this PR.

**Review by webknjaz — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -1 +1,2 @@
 # django-girls-blog-RootenberG
+[![Build Status](https://travis-ci.com/kpi-web-guild/django-girls-blog-RootenberG.svg?branch=master)](https://travis-ci.com/kpi-web-guild/django-girls-blog-RootenberG)
```
Comment (webknjaz) [inline README.md]: unrelated.

**Review by webknjaz — CHANGES_REQUESTED** [review]
Alright, now wipe all unrelated commits.

File: README.md [hunk 3]
```diff
@@ -1 +0,0 @@
-# django-girls-blog-RootenberG
```
Comment (webknjaz) [inline README.md]: unrelated

Comment (RootenberG) [reply README.md]: I deleted this file.

Comment (webknjaz) [reply README.md]: You added a commit with the file deletion. I want this change to not appear in this PR.

Comment (webknjaz) [reply README.md]: Besides, it now creates a merge conflict and you'd not be able to merge if even with approvals in place.

**Review by webknjaz — APPROVED** [review]
You can merge it if invoking it via `python3.7 -m pre_commit` locally doesn't emit any errors.
